lib/cli.js

```diff
@@ -28,6 +29,7 @@ async function main(cli, { policyDetails, setupProjectFn }) {
       let configuredPolicies = JSON.parse(configFile);
       if (configuredPolicies.primary) {
         policies.primary = configuredPolicies.primary;
+        ignoredDependencies = configuredPolicies.primary.ignoredDependencies || ignoredDependencies;
```

Comment:
looks like the config file takes precedence over the command line flag. I would think this should be the opposite

Reply:
We should probably merge and deduplicatee?

Reply:
Removed CLI option